Documentation/src/enumerable-extensions/where-not-null.swirl

```diff
@@ -0,0 +1,8 @@
+% WhereNotNull
+
+------N----2----4----N----N----1----7---|
+N:={}
```

Comment:
It's a big confusing to have `{}` stand for `null` since `{}` is also the empty property pattern which does not match null :)

I'd explicitly write `null`.

Reply:
* null would work, but the circles are pretty narrow, it looks better with {}
* The Empty-set symbol does not work, the parser ignores most of unicode, except digits and alphabetic characters.

Reply:
Unicode symbols seem to work on the left side of `:=`.
I replaced all {} with Ø which lead to an encoding problem in ths SVG (swirly.dev seems to write as ANSI encoded).

But all that should be fixed in ef91d2e1ac1a7bc4575dc815cf68d1f2e4f52cf3